pkg/printers/internalversion/describe.go

```diff
@@ -3273,7 +3273,11 @@ func printNetworkPolicySpecIngressFrom(npirs []networking.NetworkPolicyIngressRu
 		} else {
 			for _, from := range npir.From {
 				w.Write(LEVEL_0, "%s", initialIndent)
-				if from.PodSelector != nil {
+				if from.PodSelector != nil && from.NamespaceSelector != nil {
+					w.Write(LEVEL_0, "%s: %s\n         %s: %s\n",
+						"From NamespaceSelector", metav1.FormatLabelSelector(from.NamespaceSelector),
+						"PodSelector", metav1.FormatLabelSelector(from.PodSelector))
+				} else if from.PodSelector != nil {
```

Comment:
Curious why if-else-if? Why not just print every non-nil `from.Xxx`?

Reply:
I think it's because previously, you can only chose one filed from `PodSelector`/`NamespaceSelector` and `IPBlock`, can't combine them together.
Now, networkpolicy allow the combination of `PodSelector` and `NamespaceSelector`, but have different significance.

Reply:
Ok, I see what you meant now.